impl/src/test/groovy/com/okta/sdk/impl/ds/DefaultDataStoreTest.groovy

```diff
@@ -89,11 +91,13 @@ class DefaultDataStoreTest {
         def responseBody = '{"name": "jcoder"}'
         def defaultDataStore = new DefaultDataStore(requestExecutor, "https://api.okta.com/v1", apiKeyResolver)
         def requestCapture = ArgumentCaptor.forClass(Request)
+        def headers = new HttpHeaders()
+        headers.setContentType(MediaType.APPLICATION_JSON)
```

Comment:
It looks like the existing tests _should_ sill function as is without these changes.  Ideally, we don't change existing tests.

If this line is removed does the test still pass?

Reply:
No, they don't.
We need to set the "Accept" header for a response.

Reply:
small clarification:
**DefaultDataStoreTest** is mocked, this is mean that the headers from the request will not be transferred to the response. That's why we need to set them manually to the `mock(Response)`;

Reply:
@VitaliiTytarenko-okta Before this PR's change, the header used to be set by default for all requests at https://github.com/okta/okta-sdk-java/blob/master/impl/src/main/java/com/okta/sdk/impl/ds/DefaultDataStore.java#L480

This is why the tests did not require explicit setting of headers earlier. Now with this PRs change, we should not be adding explicit tests in the Unit tests and it should work without it.

Reply:
It doesn’t matter for those tests, because they are dummies. Even when the request contains the “Accept” header - the response doesn’t, because it doesn’t transfer it.
The only difference: earlier we did no do `response.getHeaders().getContentType()` (line 485 of ‘getBody’)
To return the correct response, it should now be: `MediaType.APPLICATION_JSON.equals(response.getHeaders().getContentType()) == true`
But previously was: 
```
when(response.getHeaders()).thenReturn(new HttpHeaders()) -> MediaType.APPLICATION_JSON.equals(null) == false
```